omod-2.2/pom.xml

```diff
@@ -0,0 +1,139 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.openmrs.module</groupId>
+		<artifactId>webservices.rest</artifactId>
+		<version>2.22.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>webservices.rest-omod-2.2</artifactId>
+	<packaging>jar</packaging>
+	<name>Rest Web Services 2.2 OMOD</name>
+	<description>OpenMRS module project for Rest Web Services</description>
+
+	<properties>
+		<openmrs.version.2.2>2.2.0</openmrs.version.2.2>
```

Comment:
Has openmrs platform 2.2.0 been released yet?

Reply:
not yet, I wasn't sure what to add to the pom file but I modified that to 2.1

Reply:
you would immediately get a compiler error because that version does not exist.